ocaml/xapi-idl/v6/v6_interface.ml

```diff
@@ -51,6 +51,16 @@ type edition = {
 }
 [@@deriving rpcty]
 
+(** Record representing version *)
+type version = {
+    dbv: string  (** Date-Based Version *)
+  ; preview: bool  (** In preview phase or not *)
```

Comment:
I'm sorry, but I have to do the same nitpick as on the previous PR, "preview" is not enough to understand what the field does at first glance

 ```suggestion
  ; is_prerelease: bool  (** In preview / prerelease phase or not *)
```
Adding is_ is not enough to make it clarifying enough, stating it's a prerelease version makes everything clear to me, I'd like to hear other opinions

Reply:
Thought it about for a while. I would prefer the `is_preview_release` which was suggested by @lindig. I changed it back to `preview` as I thought it would be clear under the context of version-related info.

`prerelease` seems not a release to me. But actually it is a release indeed.

Reply:
BTW, with @robhoes 's comment above, the specific name will not be hardcoded in XAPI.

Reply:
That sounds good as it releases xcp-ng from knowing about specific fields. For their purposes they can change their daemon to use the new interface and pass around any information they might want (or none at all)
@stormi